src/constants/gfx.asm

```diff
@@ -166,6 +166,7 @@ OBJECT_CLIFF_TOP            equ $4D
 OBJECT_BUSH                 equ $5C
 OBJECT_WEATHER_VANE_BASE    equ $5E
 OBJECT_WELL                 equ $61
+OBJECT_CACTUS               equ $69
```

Comment:
If cactus and spikes share the same check and object ID, maybe it makes sense to just have a "OBJECT_HURT_TILE" instead?